Gemfile

```diff
@@ -4,4 +4,5 @@ gemspec
 
 platforms :ruby do
   gem "posix-spawn"
+  gem "byebug"
```

Comment:
Did you mean to commit this?

Reply:
Nope! Also,I'm re-submitting this on terrapin shortly.